cmd/charm/charmcmd/plugin.go

```diff
@@ -100,10 +116,16 @@ func pluginHelpTopic() string {
 	return output.String()
 }
 
+// getPluginDescriptionsResults holds memoized results for getPluginDescriptions.
+var getPluginDescriptionsResults []pluginDescription
```

Comment:
perhaps call this pluginDescriptionResults getPluginDescriptionResults made me think it was a function.